ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/deprovision/cucushift-installer-rehearse-azure-ipi-deprovision-chain.yaml

```diff
@@ -5,7 +5,7 @@ chain:
   - chain: gather-core-dump
   - ref: gather-azure-resource
   - chain: ipi-deprovision
-  - ref: cucushift-installer-check-azure-destroy-dns
+  #- ref: cucushift-installer-check-azure-destroy-dns
```

Comment:
shall we change it back?

Reply:
updated.
Also updated the step to skip the check when customer dns is enabled.